### PR TITLE
fix: fixes the scroll effect

### LINKED ISSUE #822: Vertical scroll is not functional past creator review section on landing page on mobile
**Describe the bug**
On mobile, the landing page is difficult to scroll past the creators' review section. It keeps scrolling horizontally.

**To Reproduce**
1 Go to mobile landing page or click on all projects (for users not logged in)
2 Scroll down to creator reviews
3 See error 

Expected behavior
Users should be able to scroll down/up the creator review with ease to see other content in the landing page.

### PR DESCRIPTION
**Pull request by yokwejuste**
## Summary

Description of PR here...
Closes #822 

## Changes

- Team review section

## Screenshots

https://github.com/unstructuredstudio/zubhub/assets/71908316/480e7994-cf4e-4bfb-932c-015e1b553896

### CODE REVIEW

**Comment by yokwejuste** [comment]
@DonPresh here is the PR I made, waiting for @srish  and @tuxology to make a review

**Comment by tuxology** [comment]
@kamthamc Need your review here. I feel there should be a more straightforward way than hardcoded styles in `Home.jsx`. IN addition, @yokwejuste I don't understand how this is related to vertical scrolling issues. This is adding two new buttons for horizontal scrolling. We don't need them - we just need to fix vertical scrolling for now. On mobile, these can be swiped if needed (that would be an acceptable addition)

File: zubhub_frontend/zubhub/src/views/home/Home.jsx [hunk 1]
```diff
@@ -327,7 +380,37 @@ function Projects(props) {
       }}
     >
             <div className={classes.carouselContainer}>
-              <Carousel showThumbs={false} infiniteLoop={true} showStatus={false} onChange={handleSlideChange}>
+              <Carousel showThumbs={false} infiniteLoop={true} showStatus={false} onChange={handleSlideChange} swipeable={false} autoPlay={true}
```
Comment (chaitanya-cloudknox) [inline zubhub_frontend/zubhub/src/views/home/Home.jsx:383]: Can you tell me more on whats causing the scrolling issue and then how the custom arrows fix the problem?

Comment (DonPresh) [reply zubhub_frontend/zubhub/src/views/home/Home.jsx:383]: @yokwejuste @chaitanya-cloudknox I was looking at the code and I have a question. Don't you think adding `infiniteLoop`true` recreates the problem we are trying to fix?

Comment (yokwejuste) [reply zubhub_frontend/zubhub/src/views/home/Home.jsx:383]: No it doesn't, it fixes instead. The carousel package was causing it and I had to get a walk around to skip the blocked scroll

Comment (yokwejuste) [reply zubhub_frontend/zubhub/src/views/home/Home.jsx:383]: > Can you tell me more on whats causing the scrolling issue and then how the custom arrows fix the problem?

The scroll was due to the axis assigned for the scroll, so all the interaction with the carousel interface brings us to a scenario where we have all action on the carousel only linked to the axis direction.

**Comment by DonPresh** [comment]
@yokwejuste Can you pls show what it looks when you try to scroll past? The main issue here is trying to scroll past this area. How does this solve the problem? Also you omitted the image that was in the original design. Please review and make corrections.

**Comment by yokwejuste** [comment]
> @yokwejuste Can you pls show what it looks when you try to scroll past? The main issue here is trying to scroll past this area. How does this solve the problem? Also you omitted the image that was in the original design. Please review and make corrections.

Check the video attached please

**Review by tuxology — APPROVED** [review]
LGTM 🌈

**Pull request merged**